Incorporate function id in function call and remove `_load` method overloading (#52)
cognitedata/cognite-sdk-python-experimental: CHANGELOG.md

- `Security` in case of vulnerabilities.

> ## [0.7.3]

Reviewer: ```suggestion
## [0.8.1]
```

### Changed

> - Because the new function call response from the  Cognite functions api now returns functions id, the endpoint for getting logs of a call has a simplified internal structure.

Reviewer: ```suggestion
- Function calls now returns `functionId`, so the getting logs of a call has a simplified internal structure.
```
